From 904a8d31d2906e8e653d670e508c4c6bd4b264c1 Mon Sep 17 00:00:00 2001
From: Stefano Lattarini <stefano.lattarini@gmail.com>
Date: Thu, 30 Aug 2012 18:53:15 +0200
Subject: [PATCH 12/89] build: prefer pattern rules over suffix rules

That is acceptable, because the GNU parted build system already
requires GNU make anyway: the $(subst ...) built-in is used in the
common recipe for the 'ss-1024', 'ss-2048' and 'ss-4096' targets
in Makefile.am.

* include/parted/Makefile.am (.in.h.h): Rename ...
(%.h: %.in.h): ... as this pattern rule.
(SUFFIXES): Delete, no more needed.

Copyright-paperwork-exempt: yes
---
 include/parted/Makefile.am | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/include/parted/Makefile.am b/include/parted/Makefile.am
index 19236f0..6f808c6 100644
--- a/include/parted/Makefile.am
+++ b/include/parted/Makefile.am
@@ -44,8 +44,7 @@ subst_pure_attr = \
 
 $(BUILT_SOURCES): Makefile.am
 
-SUFFIXES = .in.h .h
-.in.h.h:
+%.h: %.in.h
 	$(AM_V_GEN)rm -f $@ $@-t
 	$(AM_V_at)perl -p \
 	    -e '$(subst_const_attr);' \
-- 
1.8.5.3

